xarray_sentinel/sentinel1.py

```diff
@@ -375,18 +375,60 @@ def open_pol_dataset(
     return xr.Dataset(attrs=attrs, data_vars={"measurement": arr})
 
 
-def crop_burst_dataset(pol_dataset: xr.Dataset, burst_index: int) -> xr.Dataset:
-    if burst_index < 0 or burst_index >= pol_dataset.attrs["number_of_bursts"]:
-        raise IndexError(f"{burst_index=} out of bounds")
+def find_bursts_index(
+    pol_dataset,
+    azimuth_anx_time: float,
+    use_center: bool = False,
+):
+    lines_per_burst = pol_dataset.attrs["lines_per_burst"]
+    anx_datetime = pol_dataset.attrs["sat_anx_datetime"]
+    if use_center:
+        distance = abs(
```

Comment:
I find the computation not very readable. Would you please split the steps and use descriptive names for the intermediate results?

Reply:
done